po/es.po

```diff
@@ -4550,112 +4588,118 @@ msgid ""
+"`Mutex<T>`."
 
 #: src/basic-syntax/static-and-const.md:60
 msgid ""
 "It isn't super common that one would need a runtime evaluated constant, but "
 "it is helpful and safer than using a static."
 msgstr ""
-"No es muy común que necesitemos evaluar contantes en runtime, pero ayuda y "
-"es más seguro que usando un _static_."
+"No es muy habitual que se necesite una constante evaluada en tiempo de "
+"ejecución, pero es útil y más seguro que usar una estática."
 
 #: src/basic-syntax/static-and-const.md:61
 msgid "`thread_local` data can be created with the macro `std::thread_local`."
 msgstr ""
+"Datos del `thread_local` se pueden crear con la macro `std::thread_local`."
 
 #: src/basic-syntax/static-and-const.md:63
 msgid "Properties table:"
-msgstr ""
+msgstr "Tabla de propiedades:"
 
 #: src/basic-syntax/static-and-const.md:65
 msgid "Property"
-msgstr ""
+msgstr "Propiedad"
 
 #: src/basic-syntax/static-and-const.md:65
 msgid "Static"
-msgstr ""
+msgstr "Estático"
 
 #: src/basic-syntax/static-and-const.md:65
 msgid "Constant"
-msgstr ""
+msgstr "Constante"
 
 #: src/basic-syntax/static-and-const.md:67
 msgid "Has an address in memory"
-msgstr ""
+msgstr "Tiene una dirección en la memoria"
 
 #: src/basic-syntax/static-and-const.md:67
 #: src/basic-syntax/static-and-const.md:68
 #: src/basic-syntax/static-and-const.md:70
 #: src/basic-syntax/static-and-const.md:71
 msgid "Yes"
-msgstr ""
+msgstr "Sí"
 
 #: src/basic-syntax/static-and-const.md:67
 msgid "No (inlined)"
-msgstr ""
+msgstr "No (insertado)"
 
 #: src/basic-syntax/static-and-const.md:68
-#, fuzzy
 msgid "Lives for the entire duration of the program"
-msgstr "La función `main` que es el punto de entrada al programa."
+msgstr "Vive durante toda la ejecución del programa"
 
 #: src/basic-syntax/static-and-const.md:68
 #: src/basic-syntax/static-and-const.md:69
 #: src/basic-syntax/static-and-const.md:71
 msgid "No"
-msgstr ""
+msgstr "No"
 
 #: src/basic-syntax/static-and-const.md:69
 msgid "Can be mutable"
-msgstr ""
+msgstr "Puede ser mutable"
 
 #: src/basic-syntax/static-and-const.md:69
 msgid "Yes (unsafe)"
-msgstr ""
+msgstr "Sí (inseguro)"
 
 #: src/basic-syntax/static-and-const.md:70
-#, fuzzy
 msgid "Evaluated at compile time"
-msgstr "Los valores son de tamaño fijo conocidos en tiempo de compilación."
+msgstr "Evaluado en tiempo de compilación"
 
 #: src/basic-syntax/static-and-const.md:70
 msgid "Yes (initialised at compile time)"
-msgstr ""
+msgstr "Sí (inicializado en tiempo de compilación)"
 
 #: src/basic-syntax/static-and-const.md:71
 msgid "Inlined wherever it is used"
-msgstr ""
+msgstr "Insertado dondequiera que se utilice"
 
 #: src/basic-syntax/scopes-shadowing.md:3
 msgid ""
 "You can shadow variables, both those from outer scopes and variables from "
 "the same scope:"
 msgstr ""
-"Puedes hacer seguimiento de variables, esto se puede hacer para variables "
-"dentro y fuera del scope actual:"
+"Puedes sombrear variables, tanto las de ámbitos externos\n"
```

Comment:
is this newline needed? Also, are you sure about translating "shadow" into "sombrear"? (it's already quite a few years since I coded with spanish people, I'm unsure how this would be understood)

Reply:
The /n is spurious. I guess "sombrear" should be correct, e.g., https://learn.microsoft.com/es-es/dotnet/visual-basic/programming-guide/language-features/declared-elements/shadowing